src/App.tsx

```diff
@@ -1,61 +1,41 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import { getUserById } from './services/user';
+import { Todo } from './types/Todo';
```

Comment:
in order to immediately clearly see that a type or component is being imported,
we can explicitly indicate that this is a type
`import type { Todo } from './types/Todo';`